index.html

```diff
@@ -1004,6 +1006,103 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `protocol_handlers` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+          "manifest">protocol_handlers</dfn></code> member is an <a>array</a>
+          of <a>protocol handler description</a>s that allows a web application to
+          handle URL protocols.
+        </p>
+        <p class="note">
+          Protocol handlers could, for instance, be used for web app
+          communication where one app directly invokes another and passes data
+          via custom protocol links.
+        </p>
+        <p>
+          How protocol handlers are presented, and how many of them are shown
+          to the user, is at the discretion of the user agent and/or operating
+          system.
+        </p>
+        <p>To <dfn>process the `protocol_handlers` member</dfn>, given [=object=] |json:JSON|, |manifest:ordered map|, and |manifest URL:URL|:</p>
+        <ol>
+          <li>Let <var>processedProtocolHandlers</var> be a new [=list=].
+          </li>
+          <li>[=list/For each=] <var>protocol_handler</var> (<a>protocol handler description</a>):
+            <ol>
+              <li>If |protocol_handler|["protocol"] or
+              <var>protocol_handler</var>["url"] is undefined, [=iteration/continue=].
+              </li>
+              <li>Set <var>protocol_handler</var>["url"] to the result of [=URL
+              Parser|parsing=] <var>protocol_handler</var>["url"] using
+              <var>manifest URL</var> as the base URL. If the result is 
+              failure, [=iteration/continue=].
+              </li>
+              <li>If <var>protocol_handler</var>["url"] is not
+              [=manifest/within scope=] of <var>manifest URL</var>, [=iteration/continue=].
+              </li>
+              <li>If <var>protocol_handler</var>["url"] already exists in <var>
+                processedProtocolHandlers</var>, [=iteration/continue=].
+              </li>
+              <li>
+                [=List/Append=] |protocol_handler| to
+                <var>processedProtocolHandlers</var>.
+              </li>
+            </ol>
+          </li>
+          <li>Return <var>processedProtocolHandlers</var>.
+          </li>
+        </ol>
+        <p>
+          To process the the <var>processedProtocolHandlers</var> the user
+          agent SHOULD [=register a protocol handler=] per item defined in the
+          [=sequence=].
+        </p>
```

Comment:
```suggestion
          <li>
           [=list/For each=]  |processedProtocolHandlers|, [=register a protocol handler=].
         </li>
      </ol>
```

Note that the above doesn't quite match what HTML expects.